src/components/shared/VirtualList.js

```diff
@@ -70,12 +98,27 @@ VirtualListInnerChunk.propTypes = {
   columnIndex: PropTypes.number.isRequired,
 };
 
-class VirtualListInner extends PureComponent {
+type VirtualListInnerProps = {
+  itemHeight: CssPixels,
+  className: string,
+  renderItem: RenderItem,
+  items: *[],
+  specialItems: *[],
+  visibleRangeStart: number,
+  visibleRangeEnd: number,
+  columnIndex: number,
+};
+
+class VirtualListInner extends React.PureComponent<VirtualListInnerProps> {
+  _container: ?HTMLElement;
+
+  _containerCreated(element: ?HTMLDivElement) {
+    this._container = element;
+  }
+
   constructor(props) {
     super(props);
-    this._containerCreated = e => {
-      this._container = e;
-    };
+    (this: any)._containerCreated = this._containerCreated.bind(this);
```

Comment:
nit: use a property initializer for this. Maybe rename to `_takeContainerRef` too.